Add Fazit (#11)
jneidel/internet.org-entwicklung: main.tex

In der Zwischenzeit versuchte Facebook, nach initellen Startschwierigkeiten, die Initiative durch Rebranding zu retten, was aber nicht reichte um die kritische Öffentlichkeit zu beeinflußen, deren Meinung bis zu den Entscheidungträgern der zuständigen Behörde vordrigen konnte.
Welche sich dann mit einem indirekten Verbot auf der Seite der Protestierenden positionierte.

% Fazit
\section{Fazit}

Aus unseren zusammengetragenen Informationen lässt sich erschließen, dass sich das Projekt Internet.org/Free Basic in Indien in kurzer Zeit schnell negativ entwickelt hat.
Zwischen dem formellen Start im Februar 2015 und dem Verbot im Februar 2016 ist gerade mal ein Jahr vergangen.

> Dadurch wird deutlich, dass innerhalb des Jahres viele Probleme an dem Projekt auftraten.

Reviewer: Aus unseren zusammengetragenen Informationen lässt sich erschließen, dass das Projekt Internet.org/Free Basics in Indien in kurzer Zeit (Start Februar 2015; Verbot Februar 2016) unter anderem durch negative öffentliche Meinungsäußerung gescheitert ist. Der von Facebook angezielte Markt war durch ein ähnliches und bereits unter Feuer geratenes Projekt nur Monate vorher verstimmt worden, fehlerhaftes Management, insbesondere die Stimme an Presse und Bevölkerung nahm nur selten an der Diskussion um tatsächlich besprochene Kitikpunke teil, wobei selbst dann nur das eigene Projekt verteidigt wurde. Als im September endlich nachgelassen wurde, war es bereits zu spät; die Bevölkerung stimmte der Opposition zu, was final in der Entscheidung des TRAI resultierte.

In der Zwischenzeit versuchte Facebook, nach initellen Startschwierigkeiten, die Initiative durch Rebranding zu retten, was aber nicht reichte um die kritische Öffentlichkeit zu beeinflußen, deren Meinung bis zu den Entscheidungträgern der zuständigen Behörde vordrigen konnte.
Welche sich dann mit einem indirekten Verbot auf der Seite der Protestierenden positionierte.

% Fazit
\section{Fazit}

> Aus unseren zusammengetragenen Informationen lässt sich erschließen, dass das Projekt Internet.org/Free Basics in Indien in kurzer Zeit (Start Februar 2015; Verbot Februar 2016) unter anderem durch an öffentlicher Kritik gescheitert ist.

Reviewer: ```suggestion
Aus unseren zusammengetragenen Informationen lässt sich erschließen, dass das Projekt Internet.org/Free Basics in Indien in kurzer Zeit (Start Februar 2015; Verbot Februar 2016) unter anderem an öffentlicher Kritik gescheitert ist.
```

Der von Facebook angezielte Markt war durch ein ähnliches und bereits unter Feuer geratenes Projekt nur Monate vorher verstimmt worden, fehlerhaftes Management, insbesondere die Stimme an Presse und Bevölkerung nahm nur selten an der Diskussion um tatsächlich besprochene Kitikpunke teil, wobei selbst dann nur das eigene Projekt verteidigt wurde.
Als im September endlich nachgelassen wurde, war es bereits zu spät; die Bevölkerung stimmte der Opposition zu, was final in der Entscheidung des TRAI resultierte.

> Um auf die Eingangs formulierte Frage zurückzukommen, ob der Umsetzung von Zuckerbergs Versprechen, Teile der untersten Schicht durch Zugriff auf das Internet aus der Armut zu heben, Taten gefolgt sind, lässt sich klar verneinen.

Reviewer: ```suggestion
Die anbeginns formulierte Frage, ob Facebook Zuckerbergs Versprechen, Teile der untersten Schicht durch Zugriff auf das Internet aus der Armut zu heben, nachgekommen ist, lässt sich klar verneinen.
```

> Das Projekt kann weder als Erfolg für seine Zielgruppe noch für Facebook gewertet werden. So könnte die Zielgruppe aus verschiedenen Gründen, wie mangelndem Zugriff auf Internetfähige Geräte, unzureichender Verfügbarkeit, einem schwachen Service und der Missachtung der Rolle von Frauen in der Gesellschaft, welche nicht durch kostenlosen Zugang zum Internet behoben werden können. Trotz hoher Investitionen könnte Facebook die Öffentlichkeit nicht für sich gewinnen und wurden durch die, an dem Erhalt eines offenen und fairen Internet interessierten, `Technologists' der `Save the Internet' Kampagne ausgeboten.

Reviewer: ```suggestion
Das Projekt kann weder als Erfolg für seine Zielgruppe noch für Facebook gewertet werden. So konnten die Probleme der Zielgruppe, wie mangelndem Zugriff auf internetfähige Geräte, unzureichender Verfügbarkeit, einem schwachen Service und der Missachtung der Rolle von Frauen in der Gesellschaft, nicht durch kostenlosen Zugang zum Internet behoben werden. Trotz hoher Investitionen könnte Facebook die Öffentlichkeit nicht für sich gewinnen und wurden durch die, an dem Erhalt eines offenen und fairen Internet interessierten, `Technologists' der `Save the Internet' Kampagne ausgeboten.
```
